lattice-controller/README.md

```diff
@@ -1,7 +1,40 @@
 # Lattice Controller Capability Provider
 
-A capability provider that allows actors to interact with the lattice control interface via
-the NATS substrate.
+A capability provider that allows actors to interact with the lattice control interface (`wasmcloud:latticecontrol`) by
+remotely communicating with lattices and the hosts contained within them via NATS.
 
-Build with 'make'. Test with 'make test'.
+## Configuration
+This capability provider is designed to facilitate connections to multiple lattices. For each connection that actors need to utilize, there must be an accompanying `set_lattice_credentials` call, _even if you are establishing a connection to the `default` lattice_.
 
+It may not be immediately obvious, but because of the flexible design of wasmCloud's lattice, the actor(s) that set credentials need not be the same actor(s) that utilize the established connections so long as they are all bound to the provider via empty link definitions.
+
+Further, you can run multiple instances of this provider in a source lattice (e.g. not necessarily one you're remotely managing) and it will automatically scale, with each instance maintaining its own cached connection to the appropriate lattices.
+
+## ⚠️ Compatibility Warning for versions < 0.9.0 ⚠️
+In previous versions of this capability provider, the provider would only ever establish one lattice control connection per instance, typically associated with the link name. The configuration for the lattice connection would come from the data on the link definition.
+
+This is _not_ how the current version of the provider works. The current provider is a _multiplexed_ provider supporting multiple lattices. An actor needs to establish a link once via an empty link definition, and then establish connections to remote lattices by using the `set_lattice_credentials` operation on the provider.
+
+This provider no longer supports fallback connections supplied via the provider configuration parameter at startup. In other words, you _must_ invoke `set_lattice_credentials` at least once to use this provider.
+
+
+## Actor Usage Example
+The following is an example of what it looks like for an actor to utilize this capability provider. In this sample, the actor is requesting that the `echo` actor (10 instances of it) be started in the `default` lattice:
+
+```rust
+async fn start_actor(ctx: &Context) -> RpcResult<CtlOperationAck> {
+    let lattice = LatticeControllerSender::new();
+    // NOTE: lattice credentials need to have been set for 'default' before calling this
```

Comment:
Can we show the `lattice.set_credentials("default", None, None, None)` call here, noting its for an unauthenticated local lattice?

Reply:
My goal, even if it's pretty self explanatory from the docs, is that someone can come here and take this code as-is to try it